mp-spdz-rs/README.md

```diff
@@ -2,8 +2,8 @@
 1. Run `git submodule update --init --recursive` to clone the submodules in this repo.
 2. Install `libsodium` & `gmp` via:
 ```zsh
-brew install libsodium gmp
-yum install libsodium gmp
-apt-get install libsodium gmp
+brew install libsodium gmp boost ntl
```

Comment:
do we need the `boost` package? I don't see us linking it

Reply:
we will